src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V62/Messages/BlockBodiesMessageSerializer.cs

```diff
@@ -47,9 +72,41 @@ public BlockBodiesMessage Deserialize(IByteBuffer byteBuffer)
 
         public int GetLength(BlockBodiesMessage message, out int contentLength)
         {
-            byte[] oldWay = Serialize(message);
-            contentLength = oldWay.Length;
-            return contentLength;
+            contentLength = message.Bodies.Select(b => b == null
```

Comment:
@kkokosa will LINQ have the same perf as a simple loop in this case?

Reply:
A little slower as there will be a `Select` enumerator allocated (~100 bytes) - I'd probably not bother until measured it is a problem.

Reply:
I would still move to loop